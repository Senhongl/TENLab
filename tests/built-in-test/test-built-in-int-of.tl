a = [1.0,2.2,3.9];
print(int_of(a));