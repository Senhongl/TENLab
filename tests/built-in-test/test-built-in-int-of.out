 1
 2
 3
[ CPUIntType{3} ]